connectors/source.py

```diff
@@ -684,6 +689,32 @@ def is_premium():
         """
         return False
 
+    def get_file_extension(self, filename):
+        return get_file_extension(filename)
```

Comment:
Looks redundant to me, why not use utils method directly?

Reply:
I didn't want to call the import on every source. AFAIU I would need to import it multiple times. This way it's imported once on `BaseDataSource` and I can just call back up to get to the utils func.

I agree it looks redundant. Do you think it's better coding standards to just import and call on each source directly?

Reply:
I'm on the fence here, I think it's okay to keep it and then inline if not needed.

I just feel that a method that pipes arguments to another method without any modifications is a code-smell, but IMO not a big one and okay to keep.

Reply:
I think an alternative is to not have the file extension tool in utils. It's only used in BaseDataSource anyway. In my next PR I'll just move the code to this function so it's not piping anymore.